PluginsAndFeatures/azure-toolkit-for-intellij/.idea/gradle.xml

```diff
@@ -11,12 +11,13 @@
         <option name="modules">
           <set>
             <option value="$PROJECT_DIR$" />
+            <option value="$PROJECT_DIR$/azure-sdk-reference-book" />
+            <option value="$PROJECT_DIR$/azure-toolkit-intellij-base" />
             <option value="$PROJECT_DIR$/buildSrc" />
           </set>
         </option>
-        <option name="useAutoImport" value="true" />
-        <option name="useQualifiedModuleNames" value="true" />
       </GradleProjectSettings>
     </option>
+    <option name="offlineMode" value="true" />
```

Comment:
what's 'offlineMode' for?

Reply:
to build faster by skipping updating local libs.
removed.